Packs/FeedMandiant/Integrations/FeedMandiant/FeedMandiant.py

```diff
@@ -0,0 +1,692 @@
+import demistomock as demisto
+from CommonServerPython import *  # noqa # pylint: disable=unused-wildcard-import
+
+import requests
+import traceback
+from typing import Dict
+
+requests.packages.urllib3.disable_warnings()  # pylint: disable=no-member
+
+''' CONSTANTS '''
+
+DATE_FORMAT = '%Y-%m-%dT%H:%M:%SZ'  # ISO8601 format with UTC, default in XSOAR
+INDICATORS_TYPE_RESPONSE = {'malware': 'malware',
+                            'actor': 'threat-actors',
+                            }
+MAP_TYPE_TO_URL = {
+    'Malware': 'malware',
+    'Actors': 'actor',
+    'Indicators': 'indicator'
+}
+MAP_TYPE_TO_RESPONSE = {
+    'Malware': 'malware',
+    'Actors': 'threat-actors',
+    'Indicators': 'indicators'
+}
+MAP_NAME_TO_TYPE = {
+    'Malware': ThreatIntel.ObjectsNames.MALWARE,
+    'Actors': ThreatIntel.ObjectsNames.THREAT_ACTOR
+}
+MAP_INDICATORS_TYPE = {'fqdn': FeedIndicatorType.Domain,
+                       'ipv4': FeedIndicatorType.IP,
+                       'md5': FeedIndicatorType.File,
+                       'sha1': FeedIndicatorType.File,
+                       'sha256': FeedIndicatorType.File,
+                       'url': FeedIndicatorType.URL}
+
+''' CLIENT CLASS '''
+
+
+class MandiantClient(BaseClient):
+    """Client class to interact with the service API
+    """
+
+    def __init__(self, base_url: str, username: str, password: str, verify: bool, proxy: bool, timeout: int,
+                 x_app_name: str):
+        super().__init__(base_url=base_url, auth=(username, password), verify=verify, proxy=proxy)
+        self._headers = {
+            'X-App-Name': x_app_name,
+            'Accept': 'application/json',
+            'Authorization': f'Bearer {self._get_token()}'
+        }
+        self._timeout = timeout
+
+    @logger
+    def _get_token(self) -> str:
+        """
+        Obtains token from integration context if available and still valid.
+        After expiration, new token are generated and stored in the integration context.
+        Returns:
+            str: token that will be added to authorization header.
+        """
+        integration_context = get_integration_context()
+        token = integration_context.get('token', '')
+        valid_until = integration_context.get('valid_until')
+
+        now = datetime.now(timezone.utc)
+        now_timestamp = datetime.timestamp(now)
```

Comment:
This lso includes the utc data
```suggestion
        now_timestamp = arg_to_datetime('now')
```